README.markdown

```diff
@@ -197,20 +222,69 @@ The example above demonstrates the following style guidelines:
 
 For conciseness, avoid using `self` since Swift does not require it to access an object's properties or invoke its methods.
 
-Use `self` when required to differentiate between property names and arguments in initializers, and when referencing properties in closures to make capture semantics explicit:
+Use `self` when required to differentiate between property names and arguments in initializers, and when referencing properties in closure expressions (as required by the compiler):
 
 ```swift
 class BoardLocation {
     let row: Int, column: Int
 
-    init(row: Int,column: Int) {
+  init(row: Int, column: Int) {
```

Comment:
Please either revert the 2 space indentation here or decrease with 2 spaces the entire code block.